### PR TITLE
updated gogoanime url (video->vc)

### PR DESCRIPTION
**Pull request by PsychoPast**
proof it works: 
image

### CODE REVIEW

File: websites/F/Fmovies/dist/metadata.json [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+    "fmovies.to",
+    "fmovies.wtf",
+    "ffmovies.ru",
+    "fmovies.cafe",
+    "www.fmovies.to",
+    "www.fmovies.wtf",
+    "www.ffmovies.ru",
+    "www.fmovies.cafe"
+  ],
+  "version": "1.3.8",
```
Comment (Bas950) [inline websites/F/Fmovies/dist/metadata.json:20]: version 1.0.0 on release

File: websites/G/Gogoanime/dist/metadata.json [hunk 2]
```diff
@@ -8,10 +8,11 @@
     "en": "Watch anime online in English. You can watch free series and movies online with English subtitles."
   },
   "url": [
-    "www2.gogoanime.video",
-    "gogoanime.so"
+    "gogoanime.vc",
+    "gogoanime.so",
+    ""
```
Comment (Bas950) [inline websites/G/Gogoanime/dist/metadata.json:13]: ????

File: websites/I/Invidious/dist/metadata.json [hunk 3]
```diff
@@ -32,4 +39,8 @@
     "media"
   ],
   "category": "videos"
+<<<<<<< HEAD
 }
+=======
+}
+>>>>>>> 37a38bbbf986e857696935c6b514e1d097e5ea65
```
Comment (Bas950) [inline websites/I/Invidious/dist/metadata.json:46]: maybe update fork whats going on here?

**Review by Bas950 — CHANGES_REQUESTED** [review]

**Comment by ririxidev** [comment]
This fork is not updated since you have a lot of changes that you shouldnt have. Open new pull request when you will fix it

**Pull request closed without merging**